src/Provider/ImageProvider.php

```diff
@@ -207,6 +208,49 @@ public function generatePrivateUrl(MediaInterface $media, $format)
         return $this->thumbnail->generatePrivateUrl($this, $media, $format);
     }
 
+    public function validate(ErrorElement $errorElement, MediaInterface $media)
```

Comment:
```suggestion
    public function validate(ErrorElement $errorElement, MediaInterface $media): void
```

Reply:
The class implements an interface with not signature, so we should not use return types (yet)